.github/workflows/release.yml

```diff
@@ -0,0 +1,127 @@
+name: Release Actions
+
+env:
+  BOT_USER: heroku-linguist[bot]
+  BOT_EMAIL: 136119646+heroku-linguist[bot]@users.noreply.github.com
```

Comment:
this is also related to getting signed bot commits for the GH App.  if you don't set this then the default committer will be the user who triggered the workflow. 

i like the idea of having it automatically set by https://github.com/heroku/use-app-token-action though. i believe they would have all the information to dynamically fetch this based on the APP_ID we're using.

Reply:
actually, what might be better is if these were set as organization variables.  that way [heroku/use-app-token-action](https://github.com/heroku/use-app-token-action) would not need to be modified to make extra API calls to retrieve information that is specific to our use case.

Reply:
Is the plan to switch to org variables now (and remove these hardcoded lines), or later? (I'm fine with later, it just wasn't clear since this conversation was marked as resolved without context)

Reply:
sorry, i'll get that sorted out later since it'll require filing a request against another team

Reply:
I've filed #99 to track moving these to org variables :-)